qiskit/algorithms/__init__.py

```diff
@@ -220,7 +230,7 @@
 
    eval_observables
 """
-
+from .algorithm_job import AlgorithmJob
```

Comment:
I think we should have this in the docstring part above too somewhere, otherwise, when its shown as a return type the user will be unable to navigate to it.